### PR TITLE
im reprogam achivemens

### PR DESCRIPTION
**Pull request by TheKillerA1**

### CODE REVIEW

File: source/Achievements.hx [hunk 1]
```diff
@@ -133,7 +133,7 @@ class AchievementObject extends FlxSpriteGroup {
 		if(camera != null) {
 			cam = [camera];
 		}
-		alpha = 0;
+		alpha = 1;
 		achievementBG.cameras = cam;
 		achievementName.cameras = cam;
```
Comment (LilDrippyMyFnf) [inline source/Achievements.hx:138]: stop

**Review by MemeHoovy — CHANGES_REQUESTED** [review]
https://user-images.githubusercontent.com/97995998/194797159-6e49f79d-5a9d-4adf-b4e7-c0a762213ec2.mp4

**Comment by Mastercopper2346** [comment]
Why spam

**Comment by MemeHoovy** [comment]
I'm about to report your fucking github account

**Comment by TheKillerA1** [comment]
> I'm about to report your fucking github account

Good luck :) Mon, Oct 10 7:05 pm

**Comment by ShadowMario** [comment]
https://user-images.githubusercontent.com/44785097/194932754-fe72a50e-b7a0-4d0d-9a98-f0d063b90ca1.mp4

**Pull request closed without merging**